Src/Plugins/Security/SEC101_003.GoogleApiKeyValidator.cs

```diff
@@ -42,13 +43,28 @@ static GoogleApiKeyValidator()
         }
 
         protected override IEnumerable<ValidationResult> IsValidStaticHelper(ref string matchedPattern,
-                                                                             Dictionary<string, string> groups)
+                                                                             Dictionary<string, FlexMatch> groups)
         {
+            if (!groups.TryGetNonEmptyValue("secret", out FlexMatch secret))
```

Comment:
>TryGetNonEmptyValue [](http://example.com/codeflow?start=24&length=19)

As part of this change, I continue to eliminate idionsyncratic behaviors, such as treating the initial match as a secret. Instead, we now make this explicit in the regex itself. This consistency will continue to help with mechanical/other transformations of our plugins as we update the model.